File: src/main/java/com/p2p/lending/dao/CreditlimitDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Borrowmoney;
import com.p2p.lending.pojo.Creditlimit;

public interface CreditlimitDao extends BaseDao<Object, Borrowmoney> {

	public List<Creditlimit> queryCreditlimits(Map<String, Object> map);
	
	public int insertCreditlimit(Map<String, Object> map);
	public int updateCreditlimit(Map<String, Object> map);

}

File: src/main/java/com/p2p/lending/dao/CreditlimitDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Borrowmoney
import com.p2p.lending.entity.Creditlimit

import org.springframework.stereotype.Repository

@Repository
interface CreditlimitDao : BaseDao<Any, Borrowmoney> {

    fun queryCreditlimits(map: Map<String, Any>): List<Creditlimit>

    fun insertCreditlimit(map: Map<String, Any>): Int

    fun updateCreditlimit(map: Map<String, Any>): Int

}
